DEVELOPMENT.md

```diff
@@ -78,15 +72,17 @@ _Adding the `upstream` remote sets you up nicely for regularly
 Once you reach this point you are ready to do a full build and deploy as
 follows.
 
-# Deploy core configurations and Kafka
+### Set up a Kubernetes cluster
 
-```bash
-# re-execute the script in case some errors appear. 
-# (this can happen when the CRDs hasn't been registered and we try to create a Kafka cluster)
-./test/kafka/kafka_setup.sh 
+- This guide assumes you have a Kubernetes cluster up and running. - https://kubernetes.io/docs/setup/
 
-kubectl apply -f config
-```
+1. Execute `source test/e2e-common.sh `
+1. Execute `knative_setup`
+    - This command deploys Knative Eventing.
```

Comment:
@pierDipi under the `Checking out your fork` section,  you already mention that the knative eventing repo should be cloned. so at this step, maybe it is either right to assume that the repository exist locally or add a check so that you only clone if there is no existing repo. i get a fatal error indicating that the repo already exist, but the script  continues to execute. WDYT?

Reply:
I think this is "fine", given that it doesn't break correctness. 
I didn't add a check for existence to keep the script simpler. 
Perhaps we can mention that a fatal error can occur in case you have already eventing cloned.

Reply:
sure, i think mentioning that as `NOTE` in the docs should help.

also, are there any minimum requirements for the Kubernetes cluster e.g `cpus`, `memory`, or a specific `vm-driver` to use so that the installation works seamlessly? or any cluster config will work?

Reply:
We don't have any requirements yet.